python/arcticdb/util/name_validation.py

```diff
@@ -0,0 +1,30 @@
+"""
+Copyright 2023 Man Group Operations Limited
+
+Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
+
+As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+"""
+
+from arcticdb.exceptions import ArcticDbNotYetImplemented
+
+# These chars are encoded by S3 and on doing a list_symbols they will show up as the encoded form eg. &amp
+UNSUPPORTED_S3_CHARS = {"\0", "*", "<", ">"}
+MAX_NAME_SIZE = (2**8) - 1
+
+def _check_name_validity(name, type_of_name):
+    if not len(name) < MAX_NAME_SIZE:
```

Comment:
```suggestion
    if len(name) >= MAX_NAME_SIZE:
```

Reply:
Removed this code because of a refactor, thus resolving the comment